### ISSUE TITLE
^^ (\marcato with direction #UP) gives an error

### ISSUE THREAD

**Issue by mmeyn** [jperon/lyluatex#205]
It seems like ^^ is converted to ` by LuaLaTeX:
```
\documentclass{article}
\usepackage{lyluatex}

\begin{document}
\begin{lilypond}
	{ c'4^^ }
\end{lilypond}
\end{document}
```
gives the following error:
```
(lyluatex)	Compiling score tmp-ly/a1878c6bd887636f032f2a74611f23f5 with LilyPond executable '/home/malte/LilyPond/2.19.82/bin/lilypond'.
-:30:7: Fehler: Zeichen oder Verweis nicht definiert: ``'
{{ c'4
      `}}
-:30:7: Fehler: Textzeichen ausserhalb von Textbeschriftungs- oder \lyricmode-Umgebung
{{ c'4
      `}}
schwerer Fehler: gescheiterte Dateien: "-"
```
Workaround: `^\marcato` instead of `^^`.

**Comment by jperon** [comment]
As often, a workaround is to add `[]` after `\begin{lilypond}` : 

```
\documentclass{article}
\usepackage{lyluatex}

\begin{document}
\begin{lilypond}[]
	{ c'4^^ }
\end{lilypond}
\end{document}
```

I can't figure yet how to fix all those problems with `\begin{lilypond}` alone : I think I'll have to make it clear in the doc.

**Comment by jperon** [comment]
Fixed in #206.
